tools/quantize/quantize-ort.py

```diff
@@ -55,13 +56,13 @@ def __init__(self, model_path, calibration_image_dir, transforms=Compose(), per_
 
     def check_opset(self, convert=True):
         model = onnx.load(self.model_path)
-        if model.opset_import[0].version != 11:
-            print('\tmodel opset version: {}. Converting to opset 11'.format(model.opset_import[0].version))
-            # convert opset version to 11
-            model_opset11 = version_converter.convert_version(model, 11)
+        if model.opset_import[0].version != 13:
```

Comment:
which operator in quantized crnn needs opset 13?

Reply:
@Charles-258 , please take a look.

Reply:
> which operator in quantized crnn needs opset 13?

If quant_format=QuantFormat.QDQ and per_channel=True, we need to convert onnx into opset 13.
![5MZ31C}UEF4`S SP%4H54W4](https://user-images.githubusercontent.com/68986816/192924863-75bec691-953f-45a7-9865-41b23acf7f62.png)

In other conditions, opset 11 is enough.

Reply:
Looks like the difference is the `Quantizelinear` and `DequantizeLinear` which are very important in the QDQ mode. Quantizelinear of opset 13 has added a new attribute, [more details](https://github.com/onnx/onnx/blob/main/docs/Operators.md#quantizelinear).